WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -19,16 +17,31 @@ class AccountSettingsService {
         static let pollingInterval = 60.0
     }
 
+    enum Notifications {
```

Comment:
Just a thought: (maybe) this should be a `struct`, since it has no `case`(s) ?.
On the other hand... there is no rule that says that an `enum` needs at least a `case`. Mixed feelings!